script/fetch-issues.js

```diff
@@ -19,9 +19,9 @@ console.log('Fetching issues data for', repo)
     await page.goto(`http://localhost:8080/#/r/${repo}/kanban`)
 
   await refresh()
-  await page.evaluate(token => {
-    localStorage.setItem('gh-token', token)
```

Comment:
Not storing the token means we only get limited number of request to GitHub API, which means incomplete issues.

https://deploy-preview-15--coala-gh-board.netlify.com/issues.json only has 100 issues.